openverse_catalog/dags/common/loader/reporting.py

```diff
@@ -104,10 +149,19 @@ def report_completion(
 
     # Collect data into a single message
     message = f"""
-*Provider*: `{provider_name}`
+*DAG*: `{dag_id}`
 *Date range*: {date_range}
 *Duration of data pull task*: {duration or '_No data_'}
```

Comment:
This should probably say `Duration of data pull tasks` for the reingestion workflows (rather than singular `task`)